.circleci/config.yml

```diff
@@ -174,8 +188,26 @@ jobs:
           name: deploy storybook
           command: ./bin/deploy-storybook.sh
 
+  cleanup PR-specific deploys:
+    docker:
+      - image: node:8
+    steps:
+      - run:
+          name: run nightly script
+          command: ./bin/nightly.sh
+
 workflows:
   version: 2
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 4 * * *"
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - cleanup PR-specific deploys
```

Comment:
CircleCI will run this at 0400 UTC (midnight EST) every day

Reply:
i like the cleanup strategy 👏